definitions/tools/biscuit_markdup.cwl

```diff
@@ -8,13 +8,14 @@ requirements:
     - class: ShellCommandRequirement
     - class: ResourceRequirement
       ramMin: 16000
+      coresMin: 4
     - class: DockerRequirement
-      dockerPull: "mgibio/bisulfite"
+      dockerPull: "mgibio/bisulfite:v1.3"
 
 arguments:
     ["/dev/stdout",
     { shellQuote: false, valueFrom: "|" },
-    "/usr/bin/sambamba", "sort", "-t", $(runtime.cores), "-m", "8G", "-o", "$(runtime.outdir)/markdup.bam", "/dev/stdin"
+    "/usr/bin/sambamba", "sort", "-t", $(runtime.cores), "-m", "15G", "-o", "$(runtime.outdir)/markdup.bam", "/dev/stdin"
```

Comment:
How much memory does `biscuit` itself need?  Is  the remaining ~1GB enough?

Reply:
not a ton, but to be safe, I just pushed a fix that upped the overall memory to 24G.  As soon as the checks clear, I'll merge.  Thanks!